tests/ops/functions/test_assert_valid.py

```diff
@@ -343,9 +343,7 @@ def __init__(self, x, wires):
 def create_op_instance(c, str_wires=False):
     """Given an Operator class, create an instance of it."""
     n_wires = c.num_wires
-    if n_wires == qml.operation.AllWires:
-        n_wires = 0
```

Comment:
Just curious who was using AllWires before?

Reply:
`GlobalPhase`. But doesn't really entirely make sense there anyway. `GlobalPhase` is just a special little snowflake.

Reply:
Also `SparseHamiltonian` at some point.

Reply:
Also `SparseHamiltonian` until about a year ago https://github.com/PennyLaneAI/pennylane/pull/3888

Reply:
Thanks for letting know!